.github/workflows/static.yml

```diff
@@ -35,6 +35,5 @@ jobs:
       - uses: bitovi/github-actions-storybook-to-github-pages@v1.0.3
         with:
           build_command: npm run prebuild:storybook && npm run build-storybook:v3
-          path: reservoir
+          path: "."
```

Comment:
Thanks. I think I forgot to backmerge this.

Reply:
I did not make this change. At least, I did not make it manually.